### ISSUE TITLE
Permalink Wikipedia Gallery Layer

### ISSUE THREAD

**Issue by dfa** [OpenSeaMap/online_chart#24]
This bug has been reported on the mailing list (http://sourceforge.net/mailarchive/message.php?msg_id=31210955).
###### Affected browser versions
- Seems to be a browser independent problem.
###### Description

When creating a permalink for the Wikipedia Gallery Layer, the Wikipedia POI Layer will be displayed.
###### How to reproduce the problem
- Create a permalink while Wikipedia Gallery Layer is activated.
- Reopen this link.
###### Expected result
- The Wikipedia Gallery Layer will be displayed.

**Comment by aAXEe** [comment]
Yes, this is a bug. The thumbnail feature is not a separate layer but an option for the wikipedia layer. With the current system for creating the permalinks there is no easy way of changing this.

Possible solutions:

* Implement the Wikipedia-Thumbnail Layer as extry Layer (maybe as pseudo layer)
* Change the source for generating and processing the permalinks
